gulpfile.js

```diff
@@ -120,6 +137,13 @@ gulp.task('sass', function() {
       stream: true
     }));
 });
+//
+gulp.task('doc', function() {
```

Comment:
I understand that this task (by any reason) cannot be executed as part of our _watch_, but, what do you think about executing it as part of our build task (`gulp dist`)?

If it is neither possible, you could add another step in our build triggers:

* https://github.com/FromDoppler/Doppler-UI-System/blob/master/build-w-docker.sh#L25
* https://github.com/FromDoppler/Doppler-UI-System/blob/master/package.json#L89

Reply:
In this way, the documentation will be accessible in the CDN, right?

Reply:
Please review this comment before merging.